integration_test-process.txt

```diff
@@ -15,22 +15,44 @@ After installing the test package with "zypper in"
   + registration success
     - check log
   + zypper lr has repos
+  + podman search registry-<csp>.susecloud.net returns some value
```

Comment:
Maybe it is easier to check the info is present on the system instead of installing Podman and running the command ?

Reply:
I'd say both

```
+ check `/etc/containers/registries.conf` for registry-<csp>.susecloud.net
+ read catalog `podman search registry-<csp>.susecloud.net:/`
```

Reply:
But we expect the user to not have to know about `registry-<csp>.susecloud.net` which is why we added significant amount of code to setup podman configuration. I am fine with having the search as part of the test procedure but then it should be `podman search $SOMECONTAINER`. 

That said, the problem is that the containers we currently mirror to the update infrastructure are all associated with a given subscription and at present there are no such containers for a general SLES or SLES4SAP subscription, as such the search `podman search $SOMECONTAINER` would by necessity have to be for a container that is free and on registry.suse.com. At the very least it will test that we didn't break the podman configuration.